scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model and Concepts
+
+This topic describes data models currently used in the Supply Chain Security Tools (SCST) – Artifact Metadata Repository (AMR).
+
+## <a id='overview'></a> Overview
+
+Here is a diagram that shows the data models currently used in AMR to store artifact information and the relation between them.
```

Comment:
`...used in AMR to store...` -> `...used in the AMR to store...`

Or is it supposed to be `SCST - AMR` ?

Reply:
I'm double checking with Tony on this, will update later

Reply:
it should just be `AMR`, no `SCST`